core/src/test/java/org/infinispan/protostream/domain/Account.java

```diff
@@ -19,6 +19,7 @@
    private String description;
    private Date creationDate;
    private Limits limits;
+   private Limits hard_limits;
```

Comment:
I'd use camel casing instead of ```hard_limits```, here, in toString, etc

Reply:
fixed